server/routes/getting_started/getting_started_router.ts

```diff
@@ -134,4 +129,75 @@ export function registerGettingStartedRoutes(router: IRouter) {
       }
     }
   );
+
+  router.post(
+    {
+      path: `/api/observability/gettingStarted/createAssets`,
+      validate: {
+        body: schema.object({
+          mdsId: schema.string(),
+          mdsLabel: schema.string(),
+          tutorialId: schema.string(),
+        }),
+      },
+    },
+    async (
+      context,
+      request,
+      response
+    ): Promise<IOpenSearchDashboardsResponse<any | ResponseError>> => {
+      try {
+        const { mdsId, mdsLabel, tutorialId } = request.body;
+        const fileData = await loadAssetsFromFile(tutorialId);
+
+        const objects = await createSavedObjectsStreamFromNdJson(Readable.from(fileData));
+        const loadedObjects = await objects.toArray();
+
+        const updatedObjects = loadedObjects.map((obj) => {
+          if (mdsId) {
```

Comment:
Now renames with the convention mds-(mdsId)-objectId-(originalId) to clearly distinguish the created assets. 
Sticks to the original behavior if mds is not enabled of just creating it under its original Id.